.github/workflows/C++.yml

```diff
@@ -0,0 +1,79 @@
+name: C++ Build
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  Linux:
+    runs-on: ubuntu-latest
+    container: quay.io/pypa/manylinux2014_x86_64:latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install fftw
+      run: |
+        yum install -y fftw3-devel
+
+    - name: Compile C++ code
+      run: |
+        make lib
+        make spreadtestall
+        make test
+
+  MacOS_clang:
+    runs-on: macos-latest
+    env:
+      MACOSX_DEPLOYMENT_TARGET: 10.15
```

Comment:
Is there some way to test older versions? This does seem to come up quite a lot in issues that it might be good to have this sorted out in CI.

Reply:
This is testing MINGW compilers under MSYS environment, MSYS has all the bash commands.

Reply:
But this is the macOS environment. I'm missing something here.

Reply:
I relied to a wrong thread. yes, change MACOSX_DEPLOYMENT_TARGET: 10.15 can test older versions.